commitment/tap.go

```diff
@@ -452,18 +452,14 @@ func (c *TapCommitment) Merge(other *TapCommitment) error {
 
 	// Otherwise, we'll need to merge the other asset commitments into
 	// this commitment.
-	for key, otherCommitment := range other.assetCommitments {
-		existingCommitment, ok := c.assetCommitments[key]
+	for key := range other.assetCommitments {
```

Comment:
`key` might not be the correct variable name after this change. I think it's an index now. Should we just use `i` instead?

Reply:
No, `key` is still correct IMO, since `assetCommitments` is a map, not a slice.